src/lv_core/lv_obj.c

```diff
@@ -253,6 +253,7 @@ lv_obj_t * lv_obj_create(lv_obj_t * parent, const lv_obj_t * copy)
         /*Set attributes*/
         new_obj->click        = 1;
         new_obj->drag         = 0;
+        new_obj->drag_dir     = 0;
```

Comment:
Wouldn't be better to use `LV_DRAG_DIR_ALL` here? Form `0` we can't see what it is doing now.
IMO we can enable both directions by default.

Reply:
Fixed.